sw/host/opentitanlib/src/transport/verilator/subprocess.rs

```diff
@@ -76,7 +76,7 @@ impl Subprocess {
     /// Finds a string within the verilator output.
     /// It is assumed that the [`Regex`] `re` has exactly one capture.
```

Comment:
This comment seems a bit misleading if `captures_len()` returns 2. Is there another way to phrase the constraint?

Reply:
Updated the comments.